### DISCUSSION TITLE
How to update FancyWM?

### DISCUSSION THREAD

**Discussion by javesar** [FancyWM/fancywm#202] [Q&A]
Hello, how do I update my FancyWM to the latest version? I've been tracking an issue reporting a bug that was resolved in v2.11.3 and I currently have v2.11.2

**Comment by achillesliu** [comment]
just download the release folder, unzip and copy paste all the things to your old version file location. Accept to the overwrite, configs will be kept

**Comment by javesar** [comment]
Thanks @achillesliu but I don't know where the right location is to copy the files. I found 3 possible locations on my PC:

1. C:\Users\Jonathan\AppData\Local\Microsoft\WindowsApps
2. C:\Users\Jonathan\AppData\Local\Microsoft\WindowsApps\2203VeselinKaraganev.FancyWM_9x2ndwrcmyd2c
3. C:\Users\Jonathan\AppData\Local\Packages\2203VeselinKaraganev.FancyWM_9x2ndwrcmyd2c

#1 and #2 both have the FancyWM.exe but #3 does not. When I download and extract the latest release v2.11.6 "FancyWM.Release.x64.zip" the folder contains the exe but also a bunch of dlls, jsons, pdgs, configs, etc...

**Reply by achillesliu** [reply]
what I did is to locate the folder with Start -> FancyWM -> right click -> open file location -> find the FancyWM icon (it's a shortcut) -> right click -> open file location.

Then you're in the right folder

**Comment by javesar** [comment]
Interesting, I don't have this option upon right clicking the app. I appreciate your help though
